src/index-nomodules.css

```diff
@@ -73,3 +73,7 @@ div.toolbar-pf {
 .popover {
   min-width: 120px;
 }
+
+span.fa.fa-birthday-cake.card-pf-icon-circle {
+  margin-bottom: 6px;
```

Comment:
Wouldn't it be better to enforce icon size in a wrapping element?
This seems like special-case hacking to me.
@jniederm , what do you think?

Reply:
Exactly. Enforcing the size of `div` wrapping the `img` element seems to be the way.